java/graalvm/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.2.0
```

Comment:
fine, but shouldn't we be using Artifact Registry?

Reply:
This seems to be the official location where the graalvm base image is published by oracle: https://www.graalvm.org/docs/getting-started/container-images/

Is there a way to search in Artifact Registry if GraalVM CE exists there?